### ISSUE TITLE
Lesson on using AWS pricing helper

### ISSUE THREAD

**Issue by OEDWY** [marcy-terui/gs-aws-pricing-helper#2]
Hello, 
Not very tech savvy and I don't know where or how to start using the add-on. Is there any youtube videos or other manuals that can help me understand?

Thanks,

Trevor

**Comment by rberger** [comment]
It would be good to note that the arguments to the functions should be cell addresses and not string literals. (at least that is what I found)

**Comment by marcy-terui** [comment]
Yes, this add-on helps to make your calculation sheet with cell addreses :-)
